app/models/study_json_record.rb

```diff
@@ -928,6 +928,8 @@ def outcome_analyses_data(outcome_measure, result_groups)
                                           ci_percent: StudyJsonRecord.float(analysis['OutcomeAnalysisCIPctValue']),
                                           ci_lower_limit: analysis['OutcomeAnalysisCILowerLimit'],
                                           ci_upper_limit: analysis['OutcomeAnalysisCIUpperLimit'],
+                                          ci_lower_limit_raw: analysis['OutcomeAnalysisCILowerLimit'],
```

Comment:
What's the difference between 'ci_lower_limit' and 'ci_lower_limit_raw' and also 'ci_upper_limit' and 'ci_upper_limit_raw' ? 

You probably need to change the raw values right?